src/renderer/component/publishForm/internal/channelSection.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
```

Comment:
This file no longer exists.

The file you will need to change is `component/slectChannel/view.jsx`